Ska/engarchive/fetch.py

```diff
@@ -563,6 +597,25 @@ def get_msid_data_from_server(h5_slice, filename):
 
         return (vals, times, bads, colnames)
 
+    @staticmethod
+    def _get_msid_data_from_maude(content, tstart, tstop, msid, unit_system):
+        """
+        Get time and values for an MSID from MAUDE.
+        Returned values are (for now) all assumed to be good.
+        """
+        out = maude.get_msids(msids=msid, start=DateTime(tstart).greta,
+                              stop=DateTime(tstop).greta)
+        out = out['data'][0] # but what about n>1 msids?
```

Comment:
However, a code comment explaining why the `[0]` index is being taken would be useful here since it isn't immediately obvious.